eip1191/src/main/kotlin/org.kethereum.eip1191/EIP1191.kt

```diff
@@ -0,0 +1,38 @@
+package org.kethereum.eip1191
+
+import org.kethereum.erc55.isValid
+import org.kethereum.keccakshortcut.keccak
+import org.kethereum.model.Address
+import org.kethereum.model.ChainId
+import org.komputing.khex.extensions.toNoPrefixHexString
+import java.util.*
+
+/*
+EIP-1191 Checksum as in https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1191.md
+ */
+
+private fun Address.cleanHexWithPrefix(chainId: ChainId?) =
```

Comment:
What is this function supposed to do? I think If you add a test for it you will see that this makes no sense at all ..